core/ledger/genesis/genesis.go

```diff
@@ -95,21 +80,21 @@ func MakeGenesis() error {
 				chaincode := chaincodes[name]
 				chaincodeMap, chaincodeMapOK := chaincode.(map[interface{}]interface{})
 				if !chaincodeMapOK {
-					genesisLogger.Errorf("Invalid chaincode defined in genesis configuration: %s", chaincode)
+					genesisLogger.Error("Invalid chaincode defined in genesis configuration:", chaincode)
```

Comment:
I don't understand why you would remove the `%s` in this case and change `Errorf` to `Error`. Am I missing something?